### DISCUSSION TITLE
launch train process from the script

### DISCUSSION THREAD

**Discussion by DimIsaev** [explosion/spaCy#10656] [Help: Coding & Implementations] [answered]
start the training process from the script but not SLI1 (*CLI)
 but not CLI

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @DimIsaev , not sure what you meant by SLI1, but if you want to start training from script, you can use the [train method as shown in this documentation](https://spacy.io/usage/training#api-train).

**Reply by DimIsaev** [reply]
Thank you
have already found
  that's exactly what I was looking for
